pkg/commands/workload_get_test.go

```diff
@@ -199,6 +199,61 @@ reason: OopsieDoodle
 status: "False"
 type: Ready
 
```

Comment:
Suggestion: Can you add tests that covers `spec.image` and `spec.source.image` path